.github/workflows/ci-master.yml

```diff
@@ -0,0 +1,67 @@
+name: CI-master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2-beta
+      with:
+        fetch-depth: 1
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Cache node modules
+      uses: actions/cache@v1
+      env:
+        cache-name: cache-node-modules
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+    - run: npm ci
```

Comment:
doesn't `npm ci` wipe `node_modules` before it runs?

I'm not 100% sure but I was fiddling around locally and it seems to do that.

If so, maybe we go with something like:

```sh
npm i --no-progress --no-fund --no-audit --prefer-offline
```